### PR TITLE
Added V0 & 2.4 Skirt Buttons Mods

### PR DESCRIPTION
**Pull request by xbst**
* [x] The mod, firmware configuration or slicer profile is in the correct category
    folder. Printable mods go to `printer_mods/`, firmware configurations
    go to `firmware_configurations/`, slicer profiles go to `slicer_profiles/`.
    Create a subfolder with your name, and place the mods in a subfolder with
    a descriptive name within that folder, e.g.: `/printer_mods/FHeilmann/flux_capacitor`
  * [x] Folders names MUST NOT contain spaces. If possible, make sure file names also 
    do not contain any spaces.
  * [x] For each mod, add a small `README.md` file to its folder with a short description
    of what the mod accomplishes. This readme can be used to add pictures, give assembly
    instructions or specify a bill of materials if the mod requires additional hardware.
  * [x] The PR modifies the top-level `README.md` of the category folder adding the 
    contribution to the table. Read the top part of the file for instructions on how
    to do this. Please preserve the alphabetical ordering while adding new rows. Make sure
    to fill out the compatibility matrix to indicate which versions of the Voron printer
    the submission is compatible with.
  * [x] The mod/configuration/profile has been tested by the person submitting the mod 
    and/or other Voron users. Make sure to add information about how the mod was tested below. 
  * [x] The mod is not merely a slight modification of an official Voron part, configuration 
    or profile (i.e. an official Voron part with a few mm added or removed or a slicer profile 
    which only modifies a few values). *(When in doubt, contact one of the admins in the 
    Voron discord before submitting the PR)*
  * [x] Submitted STLs are printable without support. *(If the mod does not meet this criterion
    join the Voron discord and ask the other users for advice on how to modify the mod such 
    that it does not require supports)*
  * [x] Submitted STL files are not corrupt. *(This can be tested by opening the STL in PrusaSlicer
    and checking if mesh errors are reported.)*
  * [x] Submitted STL files are oriented and scaled properly for printing.
  * [x] Submitted firmware configs or slicer profiles contain no sensitive data (e.g. API keys).

#### Which mods/configurations/profiles are added by this PR?
V0 skirt buttons
V2 skirt buttons
#### How was it tested? 
On my Vorons
#### Any background context you want to provide?
Nope
#### Screenshots (if appropriate)
No
#### Further notes
Some early versions of these files accidentally made into the repo because idiot me didn't know approved != merged. Won't happen again. Now the mods are finished and are added to the readme file.

### CODE REVIEW

**Comment by xbst** [comment]
I added M4 mount for V0 Mod too.

**Comment by xbst** [comment]
I added a pi camera mount too.

**Review by KiloQubit — COMMENTED** [review]
Thanks for your mods.

In `V0_Picam`, are `Ball.stl` and `Cam_B.STL` printable without supports?  They have steep overhangs on the bottom.

The rest looks good.

Checked:
- mods in correct directories
- no spaces in directory or file names
- there are readmes for the mods
- top level readme is updated
- STL are oriented correctly and don't require supports (except possibly the two mentioned above)

**Comment by xbst** [comment]
They are printable.

**Review by KiloQubit — APPROVED** [review]
Great, LGTM.

**Review by FHeilmann — CHANGES_REQUESTED** [review]
Thanks for your submissions.
The following files have questionable 3D printability and, while the prints may recover, I'd ask you to look into better ways to create the holes in these parts to make them more printable:
`printer_mods/xbst_/V0_Skirt_Buttons/Buttoned/A.stl`
`printer_mods/xbst_/V0_Skirt_Buttons/Buttoned/B.stl`
`printer_mods/xbst_/V0_Skirt_Buttons/Buttonless/A.stl`
`printer_mods/xbst_/V0_Skirt_Buttons/Buttonless/B.stl`

Here are some screenshots of my attempt to slice these parts with PrusaSlicer

image
image

Also, it's hard to parse where you pi0 cam mount is supposed to be attached. Could you extend your readme a little bit to describe how the ball is attached to the frame?
Bests,
-Florian

**Comment by xbst** [comment]
Fixed.

**Comment by xbst** [comment]
Fixed walls & readme.
